src/app/components/signup/signup.component.ts

```diff
@@ -51,7 +51,9 @@ export class SignupComponent extends CommonComponent{
           console.log('User not created, return code: ' + JSON.stringify(obj));
         }
 
-
+=======
+        console.debug("User Created: " + newtmpUser.username);
+>>>>>>> 82301b073acfd3901d51e37bbe48fca6348f3713
```

Comment:
And this.

Reply:
So these are all the things that I spent time fixing, and it looks like it still doesn't like what i did...  I'
m clearly not doing something correctly...